### DISCUSSION TITLE
How to remove specific contentTypes from GraphQL exporer ?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#12728] [Q&A]
I am trying to used GraphQL to export contents. However, I need to exclude some content types from the exporter.  In my startup classes I added the following

```
        services.Configure<GraphQLContentOptions>(options =>
        {
            options.ConfigureContentType("HomePage", x =>
            {
                x.Hidden = true;
                x.Collapse = true;
            });
            options.ConfigureContentType("SitePage", x =>
            {
                x.Hidden = true;
                x.Collapse = true;
            });
        });
```

But `HomePage` and `SitePage` content types are still showing up.

**Comment by ns8482e** [comment]
I have not verified but I guess it it’s    based on view permission, if you remove view permission for given user it should no show that content type in graphql
